.github/workflows/test.yml

```diff
@@ -0,0 +1,58 @@
+# Copyright The ORAS Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Tests
+
+on:
+  push:
+    branches:
+      - main
+      - release-*
+  pull_request:
+    branches:
+      - main
+      - release-*
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  test:
+    name: Test Setup ORAS CLI
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-latest]
+        version:
+          - 0.15.0
```

Comment:
Why do we add v0.15.0 to the test workflow?

Reply:
It should be `v0.15.1` and it is the last version to support ORAS artifact.

Reply:
Also, we usually test the latest 2 versions.